### PR TITLE
ISPN-7506 Tests fail after upgrading Lucene version

### PR DESCRIPTION
**Pull request by gustavonalle**
https://issues.jboss.org/browse/ISPN-7506
https://issues.jboss.org/browse/ISPN-7484

### CODE REVIEW

**Comment by Sanne** [comment]
You might want to include c00cdef96d4d02c1e7cfb9a2918fae16d0739f88

It fixes some problems with timezones, and tests failing at odd times ..

**Comment by gustavonalle** [comment]
@Sanne done!

**Comment by Sanne** [comment]
thanks! testing..

**Comment by gustavonalle** [comment]
Looking at test failures

**Comment by gustavonalle** [comment]
Should be fixed now

File: integrationtests/elasticsearch-indexmanager-it/start-elastic.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=version.elasticsearch | grep -v '\[.*\]')
```
Comment (gustavonalle) [inline integrationtests/elasticsearch-indexmanager-it/start-elastic.sh]: I did not know about ```mvn elasticsearch:run```, I'll add the the README.txt.
I also like the docker approach that makes it very easy to install new plugins

**Comment by gustavonalle** [comment]
Tests build apparently failed with disk errors:

```mv: cannot move ‘./server/integration/testsuite/target/server/node1/standalone/log/rest-access.log’ to ‘server_integration_testsuite_target_server_node1_standalone_log_rest-access.log’: No space left on device```

but it seems to have run all the tests before and it had only 3 non-related failures.

**Comment by pruivo** [comment]
looks good. pushing

**Comment by pruivo** [comment]
thanks @gustavonalle

**Comment by Sanne** [comment]
thanks @pruivo ! I couldn't test it myself but I'm happy it's merged :+1:

**Pull request merged**